layouts/partials/title.html

```diff
@@ -0,0 +1,7 @@
+<title>
+	{{~#if __docTitle}}
```

Comment:
That documentation is well confusing - "_When applied all whitespace on that side will be removed up to the first handlebars expression or non-whitespace character on that side._". What the hell does that mean?

Also, docs don't have a tilde at the end of the first `if` - might be a typo though?